src/es-features/task4.js

```diff
@@ -30,4 +30,25 @@ export function task4Old() {
 // Напишите реализацию функции task4Old на ES6+ ниже этого комментария.
 // При желании, можете использовать стрелочную функцию, вместо обычной
 
-export function task4New() {}
+export const task4New = () => {
+  const x = 10;
+  const y = 20;
+
+  const obj = {
+    x: x,
```

Comment:
if name of the variable is the same as key in an object, you can just skip ": ${var}".
`const obj = { x,  y, ...}`